src/components/Lineage/Graph/RouteWrapper/index.tsx

```diff
@@ -1,8 +1,12 @@
-import { useParams, useNavigate } from 'react-router-dom';
+import { useParams, useLocation, useNavigate } from 'react-router-dom';
 import { EgeriaLineageGraph } from '..';
 
+
 export function EgeriaLineageGraphRouteWrapper() {
-  const { guid, lineageType } = useParams();
+  const { guid } = useParams();
+  const location = useLocation();
+
+  const lineageType = location.pathname.substring(location.pathname.lastIndexOf('/') + 1);
```

Comment:
this needs a bit more attention, testing, the user might enter the url with "/" at the end, unit tests would be good to check the corner cases